analyzers/tests/SonarAnalyzer.UnitTest/TestCases/EqualityOnFloatingPoint.CSharp11.cs

```diff
@@ -5,14 +5,24 @@
 public class EqualityOnFloatingPoint
```

Comment:
From looking at the tests and implementation, I guess we don't support

```
if (f <= 3.146 && 3.146 <= f) // FN?
```
?

Could you maybe add this test to have it documented in the initial non C# 11 test cases?

I wouldn't fix it in this PR to avoid growing it too much (and it's out of scope for this issue & PR)

eventually open a separate ticket and fix it in a separate PR?

Reply:
Good catch. I added the test. If you don't mind, I will not open a ticket for this. I think the whole indirect equality check is of questionable value. It would be a better idea to have a separate rule that finds possible simplifications.